aweadmin.py

```diff
@@ -53,6 +55,50 @@ def get_jobs(baseurl,token,status):
   dumpdata(data)
   return data['data']
 
+def get_queues(baseurl,token,status):
```

Comment:
That was an artifact of a method I intended to write before I understood that client groups should be my focus. Good catch.